rooms/models.py

```diff
@@ -116,4 +118,43 @@ class Wish(models.Model):
     room = models.ForeignKey(Room, on_delete=models.CASCADE)
 
     class Meta:
-        db_table = 'wishes'
\ No newline at end of file
+        db_table = 'wishes'
+
+class RoomView(DBView, models.Model):
+    room           = models.ForeignKey(Room, on_delete=models.DO_NOTHING, related_name='rooms_view')
+    name           = models.CharField(max_length=45)
+    address        = models.CharField(max_length=200)
+    price          = models.DecimalField(max_digits=12, decimal_places=2)
+    latitude       = models.CharField(max_length=45)
+    longitude      = models.CharField(max_length=45)
+    max_people     = models.IntegerField()
+    description    = models.TextField()
+    guest_type     = models.IntegerField()
+    check_in_date  = models.DateField()
+    check_out_date = models.DateField()
+
+    view_definition = '''
+        select row_number() over() as id
+             , ms.id as room_id
+             , ms.name
+             , ms.address
+             , ms.price
+             , ms.latitude
+             , ms.longitude
+             , ms.max_people
+             , ms.description
+             , ms.guest_type
+             , ns.check_in_date
+             , ns.check_out_date
+          from rooms ms
+          left join (select room_id
+                          , max(check_in_date) as check_in_date
+                          , max(check_out_date) as check_out_date
+                       from room_reservations
+                      group by room_id) ns 
+                 on ms.id = ns.room_id
+    '''
```

Comment:
아래와 같이 SQL convention을 가져가면 더 가독성 좋을 것 같은데 어떠신가요?
`id`와 `room_id`가 겹치는 것 아닌가요?

```suggestion
    view_definition = """
        SELECT row_number() over() AS 
            id,
            ms.id room_id,
            ms.name,
            ms.address,
            ms.price,
            ms.latitude,
            ms.longitude,
            ms.max_people,
            ms.description,
            ms.guest_type,
            ns.check_in_date,
            ns.check_out_date
        FROM rooms r
        LEFT JOIN (
            SELECT 
                room_id,
                max(check_in_date) as check_in_date,
                max(check_out_date) as check_out_date
            FROM room_reservations
            GROUP BY room_id
        ) ns ON ms.id = ns.room_id
    """
```

Reply:
django로 모델링 할때 항상 id가 들어가서 먼가 왠지? 있어야 될것 같아 추가했습니다만,
빼버렸습니다~